frontends/bsd/src/app.test.tsx

```diff
@@ -444,7 +444,7 @@ test('authentication works', async () => {
   fireEvent.click(screen.getByText('6'));
 
   // 'Remove Card' screen is shown after successful authentication.
-  await screen.findByText('Remove card.');
+  await screen.findByText('Remove card', { exact: false });
```

Comment:
One way I've seen inexact matching done in the testing library docs is to use a regex: `await screen.findByText(/Remove card/)`

I would personally lean towards just checking for the actual full text that's expected on the screen, but that's just me!

Reply:
I think of the full text as copy that could change (and did change between the original screen and this PR) and I wanted to avoid testing copy. I believe this copy is final, however, so checking the actual full text makes sense to me to avoid unintended consequences of inexact matching, so I switched it.